.github/workflows/ci.yml

```diff
@@ -0,0 +1,93 @@
+name: Build and Test
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - "docs/**"
+jobs:
+  build:
```

Comment:
We could put these in the same file and then use a YAML anchor to "copy" it in, instead of duplicating. Maybe that will work?